mne/export/tests/test_export.py

```diff
@@ -227,17 +271,39 @@ def test_export_edf_annotations(tmp_path):
         ch_names=[["0"], ["0", "1"], [], ["1"]],
     )
     raw.set_annotations(annotations)
+    raw.crop(tmin)
+    assert raw.first_time == tmin
+
+    if tmin % 1 == 0:
+        expectation = nullcontext()
+    else:
+        expectation = pytest.warns(
+            RuntimeWarning, match="EDF format requires equal-length data blocks"
+        )
```

Comment:
What is this check doing? If you are checking for `tmin` to be an integer, you could also use `tmin.is_integer()`, but is this what is required to have "equal-length data blocks"?